tools/cockpit.spec

```diff
@@ -354,11 +381,18 @@ authentication via sssd/FreeIPA.
 %doc %{_mandir}/man8/pam_ssh_add.8.gz
 %dir %{_sysconfdir}/cockpit
 %config(noreplace) %{_sysconfdir}/cockpit/ws-certs.d
+%if 0%{?suse_version} > 1500
+%{_pam_vendordir}/cockpit
```

Comment:
Shouldn't be a `config(noreplace)` on suse as well? Because then making this a variable it could be the same existing line.